client/my-sites/importer/importer-header/index.jsx

```diff
@@ -61,6 +63,8 @@ class ImporterHeader extends React.PureComponent {
 			return StartButton;
 		} else if ( includes( doneStates, importerState ) ) {
 			return DoneButton;
+		} else if ( includes( errorStates, importerState ) ) {
+			return ResetButton;
```

Comment:
Am I right in thinking that it's the Reset button that is reliant on D18588-code? If so, should we break the reset/clear functionality out to a separate PR so that this can be merged independently?

Reply:
Yes. Going to merge it now, to unblock this.

Reply:
Merged in D18943-code.